src/linear_algebra/basic.lean

```diff
@@ -2686,38 +2442,6 @@ def comap_subtype_equiv_of_le {p q : submodule R M} (hpq : p ≤ q) :
 
 end module
 
-variables [ring R] [add_comm_group M] [module R M]
-variables (p : submodule R M)
-
-open linear_map
-
-/-- If `p = ⊥`, then `M / p ≃ₗ[R] M`. -/
-def quot_equiv_of_eq_bot (hp : p = ⊥) : p.quotient ≃ₗ[R] M :=
```

Comment:
Where did this go?

Reply:
I seem to be unable to remember I need to `git add`. Fixed now.